packetbeat/docs/packetbeat-reference-yml.asciidoc

```diff
@@ -0,0 +1,11 @@
+[[packetbeat-reference-yml]]
```

Comment:
Trying an experiment here. Is it worthwhile to provide this in the docs? This is sort of placeholder for a real quick reference that lists all config options.